src/app/zap-templates/templates/app/cluster-objects-src.zapt

```diff
@@ -7,6 +7,41 @@ namespace app {
 namespace Clusters {
 
 namespace detail {
+
+CHIP_ERROR FlightCheckDecodeAndEnterStruct(TLV::TLVReader & reader, TLV::TLVType & outer)
```

Comment:
can we take these out in a separate cpp file that is not code-generated?

since this file does not depend on any generator parameters, it seems that it would be useful for them to be somewhere else.

Reply:
Ideally, this function would actually be static (or in anonymous namespace), so that we don't have to worry about the symbol leaking out of this file, right?  But that would make it pretty hard to put into a non-generated file...